CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 
 ## Acceptable Contributions
 
+-Do you want to become a contributor for all solutions of problems on www.binarysearch.com?
+-Do you want to know all solutions on that site?
```

Comment:
Not "all",  but "some", even a solution is enough to contribute

Reply:
Hop 😅